com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineAsset.cs

```diff
@@ -251,6 +252,10 @@ public override Shader defaultShader
         internal List<string> beforePostProcessCustomPostProcesses = new List<string>();
         [SerializeField]
         internal List<string> afterPostProcessCustomPostProcesses = new List<string>();
+#if ENABLE_VIRTUALTEXTURES
+        [SerializeField]
+        public VirtualTexturingSettingsSRP virtualTexturingSettings;
```

Comment:
caution, public API need to be documented, I will recommend to put as much as possible as internal until we need to expose it